### DISCUSSION TITLE
"Ghost" CR  - Draft CR count is always one more than is actually there

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#312] [0. General bugs and questions]
In our one space, the count that is on the **Draft CR** tab is always one more than is actually in the list.

So when there are no draft CRs, the tab count says 1. If there is 1 draft CR, the tab count says 2.

Any idea where this "ghost" CR is coming from? I am an admin so should be able to see all of the CRs in the space.

![GitBookChangeRequestCountMismatch](https://github.com/GitbookIO/community/assets/107077736/274edf58-a4ac-4521-b3c8-8a20b116ded6)

**Comment by aikoii** [comment]
Hi @JaniceManwiller, it seems like we have a bug with Draft counters, but don't worry you are not missing any CR drafts. This should be an easy fix, although might take some time for us to get to it. I hope it's not too much of a blocker for you right now 🙏?

**Comment by JaniceManwiller** [comment]
Nope, it's not a blocker at all - just a weird thing.

Thanks,

Janice

**Reply by aikoii** [reply]
Thank you @JaniceManwiller 🙂, I totally get it! Hoping it will be fixed soon.
